dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/ConfigSpringBootApplicationClassloaderTests.java

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.springboot.support.fat;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.config.ClassloaderElement;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.config.SpringBootApplication;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.utils.HttpUtils;
+
+@RunWith(FATRunner.class)
+public class ConfigSpringBootApplicationClassloaderTests extends AbstractSpringTests {
+    @Override
+    public Set<String> getFeatures() {
+        return new HashSet<>(Arrays.asList("springBoot-1.5", "servlet-3.1"));
+    }
+
+    @Override
+    public String getApplication() {
+        return SPRING_BOOT_15_APP_BASE;
+    }
+
+    @Override
+    public AppConfigType getApplicationConfigType() {
+        return AppConfigType.SPRING_BOOT_APP_TAG;
+    }
+
+    // DO NOT OVERRIDE expectAppSuccess()
+
+    @Test
+    public void testSpringBootApplicationClassloaderDefaults() throws Exception {
+        server.stopServer(false);
+
+        ServerConfiguration sc = server.getServerConfiguration();
+        List<SpringBootApplication> configuredApps = sc.getSpringBootApplications();
+        configuredApps.clear();
+        SpringBootApplication appConfig = new SpringBootApplication();
+        appConfig.setLocation(getApplicationFile().getName());
+        appConfig.setName("testClassloaderDefaults");
+
+        List<ClassloaderElement> configuredLoaders = appConfig.getClassloaders();
+        configuredLoaders.clear();
+        ClassloaderElement loader = new ClassloaderElement();
+        configuredLoaders.add(loader);
```

Comment:
This seems to just test what putting an empty classloading element would do.  I expect that to just do the default behavior as if no classloading element was specified.  I think we should add some expanded visibility and verify that the expanded visibility is accessible from the application, something from a third-party API?  We probably should have a negative test endpoint in the `TestApplication` that is called from `CommonWebServerTests15`.  This endpoint should verify that some third-party API is not accessible from the application because when running `CommonWebServerTests15` the application should be using the default API visibility which excludes third-party API.